Three/include/CGAL/Three/Scene_item.h

```diff
@@ -285,6 +285,10 @@ public Q_SLOTS:
   void setPointsMode() {
     setRenderingMode(Points);
   }
+  //!Sets the RenderingMode to Points.
+  void setShadedPointsMode() {
+    setRenderingMode(ShadedPoints);
+  }
```

Comment:
I don't think it is possible. I tried but it broke the context menu, because this is needed by the slotName() function. I think replacing the current system of rendering mode to use setRenderingMode() instead of the specific functions would need a bigger change than this one.